<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeonWgt</class>
 <widget class="QWidget" name="LeonWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeonWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="m_pSearchEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pInfomationBtn">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>消息框</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pQuestionBtn">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>询问框</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pWarningBtn">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>警告框</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pErrorBtn">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>错误框</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="m_pCBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>文件</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>打开</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>关闭</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>保存</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>另存为</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>打印</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>退出</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>9</hour>
          <minute>10</minute>
          <second>35</second>
          <year>2017</year>
          <month>2</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="time">
         <time>
          <hour>9</hour>
          <minute>10</minute>
          <second>35</second>
         </time>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>选项卡1</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>21</number>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item row="0" column="3">
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item row="0" column="4">
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item row="1" column="3">
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item row="1" column="4">
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item row="2" column="2">
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item row="2" column="3">
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item row="2" column="4">
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item row="3" column="1">
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item row="3" column="2">
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item row="3" column="3">
             <property name="text">
              <string>19</string>
             </property>
            </item>
            <item row="3" column="4">
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>21</string>
             </property>
            </item>
            <item row="4" column="1">
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item row="4" column="2">
             <property name="text">
              <string>23</string>
             </property>
            </item>
            <item row="4" column="3">
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item row="4" column="4">
             <property name="text">
              <string>25</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>选项卡2</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget_2">
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>aa</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>bb</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>cc</string>
             </property>
            </item>
            <item row="0" column="3">
             <property name="text">
              <string>qq</string>
             </property>
            </item>
            <item row="0" column="4">
             <property name="text">
              <string>zz</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>dd</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>ee</string>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>ff</string>
             </property>
            </item>
            <item row="1" column="3">
             <property name="text">
              <string>ww</string>
             </property>
            </item>
            <item row="1" column="4">
             <property name="text">
              <string>aa</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>gg</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>hh</string>
             </property>
            </item>
            <item row="2" column="2">
             <property name="text">
              <string>ii</string>
             </property>
            </item>
            <item row="2" column="3">
             <property name="text">
              <string>asd</string>
             </property>
            </item>
            <item row="2" column="4">
             <property name="text">
              <string>xx</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>zz</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>xx</string>
             </property>
            </item>
            <item row="5" column="0">
             <property name="text">
              <string>cc</string>
             </property>
            </item>
            <item row="6" column="0">
             <property name="text">
              <string>vv</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_4</zorder>
  <zorder>tabWidget</zorder>
  <zorder>widget_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
